src/js/i18n/en-US.yml

```diff
@@ -60,6 +60,17 @@ careOptsFrontend:
               other {}
             }
           sexLabel: Sex
+    sidebar:
+      actionSidebar:
+        activityViews:
+          addedHTML: "<strong>{ first_name } { last_name }</strong> ({ role }) added this Action"
```

Comment:
I know it was a convention before to add `HTML` to i18n keys, but I no longer think it's useful.  It provides protection to use the html suffix in `templateContext` so that we're clear we're escaping user data, but in the case of i18n it's only in-house html.. the data still gets escaped.  (I think we didn't know for sure when we were first doing the i18n and making stylistic decisions)